SFUCommute/sfu_commute_V.2/SignupPage.swift

```diff
@@ -7,186 +7,118 @@
-                    //request.httpMethod = postString
-                    request.httpBody = postString.data(using: .utf8)
-                    request.setValue("application/json; charset=utf-8", forHTTPHeaderField: "Content-type")
-                    //print("I am making a request now:    ")
-                    URLSession.shared.dataTask(with: request, completionHandler: { (data, response, body) in
-                        let json = try? JSONSerialization.jsonObject(with: data!, options: .allowFragments)
-                        //print(json)
-                        //print(json["error"])
-                        let dict = json as? NSDictionary
-                        //print("This is the error::::---------------")
-                        //print(dict?["error"])
-                        //let responseString = String(data:data!, encoding: .utf8)
-                        //print(responseString)
-                        //print(body)
-                        //print(response)
-                    }).resume()
+    
+    func initTips() {
+        preferences.drawing.font            = UIFont(name: "Futura-Medium", size: 13)!
+        preferences.drawing.foregroundColor = UIColor.white
+        preferences.drawing.backgroundColor = Colors.SFUBlue
+        preferences.drawing.arrowPosition   = EasyTipView.ArrowPosition.bottom
+        EasyTipView.globalPreferences       = preferences
+    }
+    
+    func forData() {
+        let emailText     =  String(email.text!)
+        let passwordText  =  String(password.text!)
+        let firstNameText =  String(firstName.text!)
+        let lastNameText  =  String(lastName.text!)
+        
+        if let url = URL(string: "http://54.69.64.180/signup") {
+            var request        = URLRequest(url: url)
+            request.httpMethod = "POST"
+            var postString     = "{\"email\": \""
+            postString        += emailText!
+            postString        += "\", \"password\": \""
+            postString        += passwordText!
+            postString        += "\", \"firstname\" : \"" + firstNameText!
+            postString        += "\", \"lastname\":\"" + lastNameText! + "\"}"
+            request.httpBody   = postString.data(using: .utf8)
+            request.setValue("application/json; charset=utf-8", forHTTPHeaderField: "Content-type")
+            URLSession.shared.dataTask(with: request, completionHandler: { (data, response, body) in
+                let json = try? JSONSerialization.jsonObject(with: data!, options: .allowFragments)
+                print(json)
+                //print(json["error"])
+                let dict    = json as? NSDictionary
+                let success = String(describing: dict!["success"])
+                //let boolValue = Bool(success)
+                print("bool value is causing problem:----?")
+                print(success)
+                if (success == "Optional(1)"){
```

Comment:
Here is what I was talking about, the reason to use **SwiftyJSON**..

I will approve this PR, @jyotsnajaswal can merge it, and I will finish up sign up styling.

Thx